src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -354,15 +354,16 @@ const supplierInvoicesInitialiser = () => {
  * @returns  {object}
  */
 const supplierRequisitionInitialiser = requisition => {
-  const { program, items: backingData } = requisition;
+  const { isFinalised, program, items: backingData } = requisition;
 
   const usingPrograms = !!program;
   const route = program ? ROUTES.SUPPLIER_REQUISITION_WITH_PROGRAM : ROUTES.SUPPLIER_REQUISITION;
 
   const sortedData = backingData.sorted('item.name').slice();
-  const filteredData = usingPrograms
-    ? sortedData.filter(item => item.isLessThanThresholdMOS)
-    : sortedData;
+  const filteredData =
+    !usingPrograms || isFinalised
```

Comment:
I'm being pedantic now - this is not a "must do" - i'm approving anyway. Just a suggestion: Because we're using `!usingPrograms || isFinalised` twice, assigning it to a variable reduces the duplication and also makes this more self documenting:

i.e.: `const shouldShowOverStockItems = !usingPrograms || isFinalised` (too long of a name? 🤣 )

Reply:
Great suggestion and I agree with you. Regarding the name, maybe removing `should` still makes sense and it is not too long. ( `const showOverStockItems = !usingPrograms || isFinalised`)
Thanks for your feedback, cheers!